DIPs/DIP1NNN-CrazyPython.md

```diff
@@ -0,0 +1,167 @@
+# Dynamic-size static arrays for D
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0 (edited by DIP Manager)                                       |
+| Author:         | James Lu (jamtlu@gmail.com)                                     |
+| Status:         | Draft                                                           |
+
+## Abstract
+Often, it is necessary to allocate a data structure on the stack based on a
+size known at runtime. This proposal suggests adding dynamic-size static arrays,
+which are destroyed when they go out of scope. Dynamic-size static arrays and
+slices of them may be passed up the stack. The syntax is designed to encourage
```

Comment:
You probably mean `down the stack`, as in the majority of cases, the stack grows down ?